framework/transport/ieee1905_transport/ieee1905_transport_local_bus.cpp

```diff
@@ -81,7 +81,7 @@ void Ieee1905Transport::handle_local_bus_cmdu_tx_message(CmduTxMessage &msg)
         }
 
         if (msg.metadata()->msg_type != ntohs(ch->messageType)) {
-            MAPF_WARN("CmduTxMessage messageType mismatch.");
+            MAPF_WARN("CmduTxMessage messageType mismatch: cmdu: " << std::hex << msg.metadata()->msg_type << " packet: " << ntohs(ch->messageType));
```

Comment:
(nitpick) if it's hex, please add 0x.

Reply:
This log:

> Received ACK_MESSAGE, mid=0x8000

Was created by this code
`LOG(DEBUG) << "Received ACK_MESSAGE, mid=" << std::hex << int(mid);`

On the other hand, the [standard](https://en.cppreference.com/w/cpp/io/manip/hex) says nothing about auto adding 0x. So I guess LOG does it.
But yet on another hand, you asked to add 0x.
All in all - I'm confused
:-)

Reply:
I was confused for a while as well... But it turns out that easylogging++ already does a `showbase` on the stream, so no need for an explicit `0x`. Sorry for the confusion!